classes/ObjectModel.php

```diff
@@ -1876,15 +1876,28 @@ public function deleteImage($force_delete = false)
      *
      * @return bool
      */
-    public static function existsInDatabase($id_entity, $table)
+    public static function existsInDatabase($id_entity, $table = null)
     {
+        if ($table !== null) {
```

Comment:
Hi @PululuK based on the initial issue, this will not fix the problem when calling `Order::existsInDatabase(1, 'orders');` because it will still fail with the `orders` parameter since `id_orders` does not exist

I wonder if we could not use the definition at all times and make `$table` deprecated? Or the table could be used in case the definition isn't correctly set like if you try to use `ObjectModel::existsInDatabase(1, 'product');` then you can only rely on the table parameter.

Reply:
Hi @jolelievre  
Thanks for your recommendations... it is done !
 Can you take a look please?

Reply:
ping @jolelievre